src/de/espend/idea/laravel/controller/ControllerReferences.java

```diff
@@ -62,6 +63,8 @@
         new MethodMatcher.CallToSignature("\\Illuminate\\Routing\\Router", "group"),
     };
 
+    private static ControllerCollector controllerCollector = new ControllerCollector(new LaravelControllerNamespaceCutter());
```

Comment:
this is a common DIC pattern, but to make in really usable in the java / intellij extension system a static call is not possible. basically the extension system will provide a list of all implementations, so in this case it will then provide `Collection<ControllerNamespaceCutter>`. maybe move and Cutter Instance into `visitControllerActions`